src/helpers/hibernation.ts

```diff
@@ -0,0 +1,10 @@
+import db from './mysql';
+
+export default function hibernate(id: string, action: string) {
+  if (!id || !action) throw new Error(`missing params. 'id' and 'action' required`);
+  if (!['hibernate', 'reactivate'].includes(action)) throw new Error('invalid action');
+
+  const query = `UPDATE spaces SET hibernating = ? WHERE id = ? LIMIT 1`;
```

Comment:
We should use `hibernated` name, similar than `created` or `updated`
```suggestion
  const query = `UPDATE spaces SET hibernated = ? WHERE id = ? LIMIT 1`;
```